recipe/meta.yaml

```diff
@@ -23,7 +23,11 @@ requirements:
 
 test:
   requires:
-    - python {{ python_min }}
+    # The linter will complain about the missing python_min, but we need
```

Comment:
Question, since I had a need to ignore the linter elsewhere already: is there a way to tell the linter "this is intended"? Something like the `# type: ignore` for Mypy or `# noqa:` for Ruff?

Reply:
Nothing structured like that, because smithy is just string-parsing, mainly (things should get easier with the v1 recipe).

However, Isuru recently introduced a [mechanism](https://github.com/conda-forge/conda-smithy/pull/2090) opt out of certain lints, which could be extended to `python_min`

Reply:
See https://github.com/conda-forge/conda-smithy/issues/2170